### PR TITLE
[Workers] Bindings page update

### PR DESCRIPTION
**Pull request by deadlypants1973**
do not merge

### CODE REVIEW

File: content/workers/platform/bindings/_index.md [hunk 1]
```diff
@@ -7,14 +7,41 @@ title: Bindings
...
+Bindings can be configured one of two ways:
+
+* Updating your project's [`wrangler.toml` file](/workers/wrangler/configuration/#bindings).
+* Logging in to the [Cloudflare dashboard](https://dash.cloudflare.com) > Account Home > [**Workers**](https://dash.cloudflare.com/?zone=workers) > your **Worker** > **Settings**> **Variables**.
+
+### Service bindings
+
+Service bindings allow for communication with another Worker.
+
+To configure Service bindings via your `wrangler.toml` file, refer to [About Service bindings](/workers/platform/bindings/about-service-bindings/).
```
Comment (WalshyDev) [inline content/workers/platform/bindings/_index.md]: We talk about setting in wrangler.toml, makes sense to link to that part
```suggestion
To configure Service bindings via your `wrangler.toml` file, refer to [About Service bindings](/workers/platform/bindings/about-service-bindings/#wrangler).
```

File: content/workers/platform/bindings/_index.md [hunk 2]
```diff
@@ -7,14 +7,41 @@ title: Bindings
...
-Bindings can be configured [via Wrangler](/workers/wrangler/configuration/#bindings) using `wrangler.toml` or by logging in to the Cloudflare Dashboard > Account Home > [**Workers**](https://dash.cloudflare.com/?zone=workers) > your **Worker** > **Settings**> **Variables**.
\ No newline at end of file
+Bindings can be configured one of two ways:
+
+* Updating your project's [`wrangler.toml` file](/workers/wrangler/configuration/#bindings).
+* Logging in to the [Cloudflare dashboard](https://dash.cloudflare.com) > Account Home > [**Workers**](https://dash.cloudflare.com/?zone=workers) > your **Worker** > **Settings**> **Variables**.
+
+### Service bindings
+
+Service bindings allow for communication with another Worker.
```
Comment (WalshyDev) [inline content/workers/platform/bindings/_index.md:21]:
```suggestion
[Service bindings](/workers/platform/bindings/about-service-bindings/) allow for communication with another Worker.
```

File: content/workers/platform/bindings/_index.md [hunk 3]
```diff
@@ -7,14 +7,44 @@ title: Bindings
...
-5. [Queue bindings](/queues/configuration/) for communication between a Worker and a Queue.
+There are multiple types of bindings available today.
 
 ## Configuration
 
-Bindings can be configured [via Wrangler](/workers/wrangler/configuration/#bindings) using `wrangler.toml` or by logging in to the Cloudflare Dashboard > Account Home > [**Workers**](https://dash.cloudflare.com/?zone=workers) > your **Worker** > **Settings**> **Variables**.
\ No newline at end of file
+Bindings can be configured one of two ways:
+
+* Updating your project's [`wrangler.toml` file](/workers/wrangler/configuration/#bindings).
```
Comment (dcpena) [inline content/workers/platform/bindings/_index.md:16]: For this item and the next, I'd swap out the verbs to use be simple present, so Update and Log in.

**Review by dcpena — APPROVED** [review]

File: content/workers/platform/bindings/_index.md [hunk 4]
```diff
@@ -7,14 +7,44 @@ title: Bindings
...
-3. [R2 bucket bindings](/r2/data-access/workers-api/workers-api-reference/#create-a-binding) for communication between a Worker and an R2 bucket.
-4. [Durable Object bindings](/workers/runtime-apis/durable-objects/#accessing-a-durable-object-from-a-worker) for communication between a Worker and a Durable Object.
-5. [Queue bindings](/queues/configuration/) for communication between a Worker and a Queue.
+There are multiple types of bindings available today.
 
 ## Configuration
 
-Bindings can be configured [via Wrangler](/workers/wrangler/configuration/#bindings) using `wrangler.toml` or by logging in to the Cloudflare Dashboard > Account Home > [**Workers**](https://dash.cloudflare.com/?zone=workers) > your **Worker** > **Settings**> **Variables**.
\ No newline at end of file
+Bindings can be configured one of two ways:
```
Comment (deadlypants1973) [inline content/workers/platform/bindings/_index.md]:
```suggestion
Bindings can be configured by one of two ways:
```

**Pull request merged**